src/bp/nlu/stan/train-service.ts

```diff
@@ -37,7 +36,7 @@ export default class TrainService {
     }
 
     try {
-      const trainSet: sdk.NLU.TrainingSet = {
+      const trainSet: nluEngine.TrainingSet = {
```

Comment:
I don't know why but I'd rather like NLUEngine  with a capital N for typings ... but that's probably my autism ..